site/en/guides/porting/index.md

```diff
@@ -27,7 +29,10 @@ OpenThread requires the following platform services:
 Several example builds are provided in the OpenThread repository. For more
 information, see [Platforms](https://openthread.io/platforms).
 
+
+> *TODO* : There really isn't an easy way to show this example for the broken-out vendor repos
```

Comment:
Assuming a vendor ports OpenThread to their platform, is there a desire or a pathway for them to have the `openthread` organization fork their repo?

Reply:
We are happy to host a vendor/platform repo in the OpenThread org. I think we can mention that as an option.

Reply:
@jwhui can you provide the correct info to include on how someone would go about requesting that a vendor repo be created?

Reply:
Posting an issue at https://github.com/openthread/openthread/issues should be sufficient.